src/js/commands/divideCommand.js

```
@@ -0,0 +1,17 @@
+export default class DivideCommand {
+  constructor(value) {
+    this.value = value;
+  }
+
+  execute(currentValue) {
+    if (currentValue !== 0) {
+      return this.value / currentValue;
+    } else {
+      return 'Деление на 0!'
```

As the message returned can be specific to UI it probably better to handle it on higher level.
You can throw an error here and then catch it later.

---

Implemented error handling in Calculator instead of returning string from command.